.gitignore

```diff
@@ -0,0 +1,3 @@
+target/
```

Comment:
Yes, I'll create a separate PR for this. Or is there a reason that you did not have a `.gitignore`?